pyseir/cli.py

```diff
@@ -53,40 +54,37 @@ def _run_infer_rt(states: List[str], states_only=False):
         infer_rt.run_rt_for_fips(fips)
 
 
-def _run_mle_fits(states: List[str], states_only=False):
-    for state in states:
-        model_fitter.run_state(state, states_only=states_only)
-
-
-def _run_ensembles(states, ensemble_kwargs=dict(), states_only=False):
-    for state in states:
-        ensemble_runner.run_state(state, ensemble_kwargs=ensemble_kwargs, states_only=states_only)
-
-
 def _map_outputs(
-    states, output_interval_days=1, states_only=False, output_dir=None, run_mode="default"
+    states: List[pipeline.Region],
+    output_interval_days=1,
+    states_only=False,
+    output_dir=None,
+    run_mode="default",
 ):
     web_ui_mapper = WebUIDataAdaptorV1(
         output_interval_days=output_interval_days, run_mode=run_mode, output_dir=output_dir,
     )
     for state in states:
-        web_ui_mapper.generate_state(state, whitelisted_county_fips=[], states_only=states_only)
+        state_input = pipeline.RegionalWebUIInput.from_region(state)
+        web_ui_mapper.generate_state(
+            state_input, whitelisted_county_fips=[], states_only=states_only
+        )
 
 
 def _state_only_pipeline(
-    state, run_mode=DEFAULT_RUN_MODE, output_interval_days=1, output_dir=None,
+    state: pipeline.Region, run_mode=DEFAULT_RUN_MODE, output_interval_days=1, output_dir=None,
```

Comment:
I'm a bit hesitant to intermix `state` as a string vs `state` as a region.  It's currently a consistent pattern that all states are 2 letter abbreviations.  I'd prefer for it to be explicit between those two until everything is consistent.  Thoughts?

Reply:
My preference might be to name the variable `region` and then assert that it's a state level region at the beginning of this function.

Reply:
I was thinking that in places where the type is represented in Python types (here `region: Region`) I can immediately start exercising some liberty with the var name. But I don't care much and asserts work too, so added one and renamed.